homeassistant/components/touchlinesl/coordinator.py

```diff
@@ -0,0 +1,57 @@
+"""Define an object to manage fetching Touchline SL data."""
+
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+from pytouchlinesl import Module, Zone
+from pytouchlinesl.client import RothAPIError
+from pytouchlinesl.client.models import GlobalScheduleModel
+
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class TouchlineSLModuleCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class TouchlineSLModuleCoordinator(DataUpdateCoordinator[..]):
```
Add the return type of `_async_update_data`